src/msp_displayport.c

```diff
@@ -1055,6 +1055,11 @@ void parse_vtx_params(uint8_t isMSP_V2) {
         return;
     }
 
+    // Check for unknown VTX device  (probably from a device that is not interested in controlling the VTX directly)
+    if (msp_rx_buf[0] = 0xff) {
```

Comment:
Assingment is intentional?

Reply:
Oops. Thanks, I’ll fix it.

Reply:
Would be nice to have consts on plece :)

Reply:
Done.

Reply:
Much better now 👌